qemu/tests/cfg/blockdev_inc_backup_add_bitmap_to_raw.cfg

```diff
@@ -5,7 +5,6 @@
 
 - blockdev_inc_backup_add_bitmap_to_raw:
     only Linux
-    only filesystem iscsi_direct ceph nbd gluster_direct
```

Comment:
we'd better remove the first 2 lines too, which is the description for storage backends
please also remove them from your two other PRs

Reply:
Have updated all, thanks.